packages/documentation/scripts/types/AllFilenames.d.ts

```diff
@@ -57,8 +57,8 @@ export type AllDocsPages =
   | "reference/Advanced Types.md"
   | "reference/Declaration Merging.md"
   | "reference/Decorators.md"
-  | "reference/ESM Support for Node.md"
   | "reference/Enums.md"
+  | "reference/ESM Support for Node.md"
```

Comment:
I honestly have no idea why these files are still changed after I merged #2900. Probably best to drop them from the PR and limit this to the package.json and lockfile updates, although my impression is that even if we fix this one thing, you get an even more cryptic issue later in non-node-16 anyway so...